From eef42092059d2d6d296c91b31c258cf61786f2c5 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Wed, 27 Feb 2019 00:30:41 +0200
Subject: [PATCH] PackageInstaller: Fix AAPT warnings

Fixes:
res/layout-watch/wear_review_permission_title_pref.xml:23: warn:
generated id 'android:id/icon' for external package 'android'.
res/layout-watch/wear_review_permission_action_pref.xml:17: warn:
generated id 'android:id/title' for external package 'android'.

Change-Id: I6a909f73fb9273026e4013e8099ff346e58a81f3
---
 res/layout-watch/wear_review_permission_action_pref.xml | 2 +-
 res/layout-watch/wear_review_permission_title_pref.xml  | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/res/layout-watch/wear_review_permission_action_pref.xml b/res/layout-watch/wear_review_permission_action_pref.xml
index 2b10c4db..5970c0aa 100644
--- a/res/layout-watch/wear_review_permission_action_pref.xml
+++ b/res/layout-watch/wear_review_permission_action_pref.xml
@@ -16,7 +16,7 @@
 
 <TextView
         xmlns:android="http://schemas.android.com/apk/res/android"
-        android:id="@+android:id/title"
+        android:id="@android:id/title"
         android:gravity="center"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
diff --git a/res/layout-watch/wear_review_permission_title_pref.xml b/res/layout-watch/wear_review_permission_title_pref.xml
index 67928356..3a0b590a 100644
--- a/res/layout-watch/wear_review_permission_title_pref.xml
+++ b/res/layout-watch/wear_review_permission_title_pref.xml
@@ -20,7 +20,7 @@
         android:layout_height="wrap_content"
         android:paddingBottom="@dimen/wear_permission_review_pref_padding"
         android:orientation="vertical">
-        <ImageView android:id="@+android:id/icon"
+        <ImageView android:id="@android:id/icon"
                    android:adjustViewBounds="true"
                    android:maxHeight="@dimen/wear_permission_review_icon_size"
                    android:maxWidth="@dimen/wear_permission_review_icon_size"
@@ -28,7 +28,7 @@
                    android:layout_width="wrap_content"
                    android:layout_height="wrap_content"
                    android:scaleType="centerInside" />
-        <TextView android:id="@+android:id/title"
+        <TextView android:id="@android:id/title"
                   android:gravity="center"
                   android:layout_width="match_parent"
                   android:layout_height="wrap_content"
-- 
2.17.1

